main.go

```diff
@@ -58,12 +73,20 @@ func main() {
 		metricsPort = fmt.Sprintf(":%d", v)
 	}
 
+	managementConfig, workloadConfig, err := createClientConfigs(managementKubeConfigPath, workloadKubeConfigPath)
+	if err != nil {
+		klog.Fatalf("Can't set client configs: %v", err)
+	}
+
+	managementClient, workloadClient, err := createClients(managementConfig, workloadConfig)
+	if err != nil {
+		klog.Fatalf("Can't create clients: %v", err)
+	}
+
 	// Create a new Cmd to provide shared dependencies and start components
 	klog.Info("setting up manager")
-	syncPeriod := 10 * time.Minute
-	mgr, err := manager.New(control.GetConfigOrDie(), manager.Options{
+	mgr, err := manager.New(managementConfig, manager.Options{
```

Comment:
manager's config here should be workloadConfig? as it will be used for watching csr updates